### PR TITLE
JUCX: specify path for error file.

### LINKED ISSUE #3737: segfault in java benchmark
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/label=hpc-test-node3,worker=0/10729/console
```
18:13:52 + for iface in '$ifaces'
18:13:52 + '[' -n ib0 ']'
18:13:52 ++ ip addr show ib0
18:13:52 ++ awk '/inet /{print $2}'
18:13:52 ++ awk -F / '{print $1}'
18:13:52 + server_ip=11.151.71.0
18:13:52 + '[' -z 11.151.71.0 ']'
18:13:52 + echo 'Running standalone benchamrk on ib0'
18:13:52 Running standalone benchamrk on ib0
18:13:52 + java_pid=19415
18:13:52 + sleep 10
18:13:52 + java -cp bindings/java/src/main/native/build-java/jucx-1.7.0.jar org.ucx.jucx.examples.UcxReadBWBenchmarkReceiver s=11.151.71.0 p=20009
18:13:53 Waiting for connections on /11.151.71.0:20009 ...
18:14:03 + java -cp bindings/java/src/main/native/build-java/jucx-1.7.0.jar org.ucx.jucx.examples.UcxReadBWBenchmarkSender s=11.151.71.0 p=20009 t=10000000
18:14:04 [hpc-test-node3:24568:0:24574] Caught signal 11 (Segmentation fault: invalid permissions for mapped object at address 0x7ffff7ff7780)
18:14:04 ==== backtrace (tid:  24574) ====
18:14:04  0 0x00000000006db7f9 jni_ReleasePrimitiveArrayCritical()  ???:0
18:14:04  1 0x00000000000034f3 Java_java_util_zip_Inflater_inflateBytes()  ???:0
18:14:04 =================================
18:14:04 Sending notification to yosefe@mellanox.com
18:14:09 [hpc-test-node3:24568:0:24574] Process frozen...
```

### PR DESCRIPTION
**Pull request by petro-rudenko**
## What
specify the path for coredump java files to analyze.

## Why ?
To be able to figure out an issue in #3737

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10744/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 28-Jun-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/df1d1998de27e0a0e6b82ca312c4273bf141d873/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/43c8c1f21b753856e01487cace3d1aab015951d0/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/360df701e7f9d4460db2c48c26eb3c37b87e4f0d/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/7d8f2e350caf6aac3af0baa87ff3e5ee088c21aa/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/e723773424672b1535f73e2fb56c6ec9ac268a31/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/6ed32f3be6ebcdbe6a2f55cbf9e98fbf1b422c16/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/f111c110a8db771eb365c7506c54be8ce193bfd1/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/2c0be8dd34b7130936f89b4094bdec746b356545/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/f205ef57038692fcd6be9bad3db6634a6b3e32d1/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/c3e82213a49b43c6e1da09ca2baad75fb5d3c55d/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/6ee4eaba08bb47f3cdb99f2aea396a910bd00313/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/944a2421279d944ff799dbf9b3c919c382597b8d/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/4242c017b5200b159930c18df7e08d169e548102/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W0](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/e81d9adc7fa2a9966116476e7c83173590c825dd/hpc-test-node2_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W1](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/8dbf278cd16fb88e4338b56125c9d9aa0e69ac44/hpc-test-node2_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W2](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/a3ab4bd1766217fc462275422d9ce9a451407b07/hpc-test-node2_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W3](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/5dfe824f70824ed90ac250f7cf729e7aebfa5bb2/hpc-test-node2_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W0](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/ac69d18ba740cddead6ef615608b607f24840690/hpc-test-node3_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W1](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/78ae78868422826d825ee921cfe78aa040fdc60a/hpc-test-node3_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W2](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/bdb345adb7ca1f7254e5b4b7c0b8148a46907bd3/hpc-test-node3_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W3](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/98790142822e8e38653e870225cf015b3afd00b0/hpc-test-node3_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node_W0](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/ed51295ac2c6802014b3188db9c721c3d8d76adc/hpc-test-node_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node_W1](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/8bcc6abfdf9b086b62161ed222320d3ba2d1b57c/hpc-test-node_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node_W2](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/0a814d1d5aebb0cfc99d9f0ab475d69bab9634a1/hpc-test-node_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node_W3](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/1f0014c11467dc3ff3b55905e0a2f5e86e1ffec4/hpc-test-node_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/de534ad22188f8c915ec3a0ae3c39a183f1d8806/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/a7a3e898707b0d9fc87fdabff51fb10f30b25540/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/e8e51996a8a02c5c972c13cab37d3c7a9401a149/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/21f533dee9e8d50d10ceb3af5127cd8d89d1d117/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/89772c35c9540898dfa1136418dce00dbce85006/vulcan-jenkins_W0) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/a30a4bd85ac2b3d97662b4da17d0b1469080ea0c/vulcan-jenkins_W1) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/2232d2f8cae24c368dd58f1591cfc3f00fcd0262/vulcan-jenkins_W2) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/fa63e7c4946d9b7f4a81a812a61a2480/raw/5ed61cb573814bfaa6bbc0e5a9d9035676a2470c/vulcan-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
bot:pipe:retest

**Comment by yosefe** [comment]
bot:pipe:retest

**Comment by yosefe** [comment]
bot:pipe:retest

**Comment by yosefe** [comment]
bot:pipe:retest

**Comment by yosefe** [comment]
bot:pipe:retest

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -979,12 +979,14 @@ test_jucx() {
                         fi
                         echo "Running standalone benchamrk on $iface"
 
-                        java -cp bindings/java/src/main/native/build-java/jucx-*.jar \
+                        java -XX:ErrorFile=/scrap/core_files/java_$BUILD_NUMBER_pid%p.log \
```
Comment (yosefe) [inline contrib/test_jenkins.sh]: can we avoid system dependency?
maybe read sysctl kernel.core_pattern?
BTW which kind of file is it? is it a textual log file, or core dump file?

Comment (yosefe) [reply contrib/test_jenkins.sh]: need to pack the java log file as Jenkins build artifact

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10813/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 02-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/0d2b25da4ef8c669ee46c20cd3ec2a209abac69a/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/5582bf37e2f83e54c8eb8e7443dba85a50a35929/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/6cba984ece372920284f01ae53c00407c7092c1b/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/51a6477b8f64a390c84f1c2a6530c800ac50887c/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/5036c3db8cf6af289c66d9f278863c50d7d91e24/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/59c34e1feef5aace60b5384d70f5e2d3a8c23477/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/f5540bb4f1952940e1fa963856c665c102229eaf/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/3ec926c3f96c2e0ca7f934e2100115b4ec1fd05b/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/79da840c15dab192aaefdb276e1c611f3a1f2404/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/1598481dd677a1fc12bd767926305b1b62994a21/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/a41aeb4a3abef9ff076edfa484a5d9424914e803/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/298200497d066650315685764f4641b221c7eab1/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/eb3f5900dc7017ff80f1de428d6322ccd5363494/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W0](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/dad026f44090b4ea7021b5b3bf2a936ec1b6d828/hpc-test-node2_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W1](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/a8dd650e244de4b5fdc27ce6c5b268fc79567809/hpc-test-node2_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W2](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/6a2905728fa45cfc2479aa674e4fbbfd8160b1c6/hpc-test-node2_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W3](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/2f83c156643b5cf5fff2ddea32d80de9fd1157d4/hpc-test-node2_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W0](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/82d117e8a20a105cb9e62d75ce50353687815933/hpc-test-node3_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W1](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/441b6e648d493ac98c0a8b7bf05bf664e098da51/hpc-test-node3_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W2](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/e6c4064c053f5296d9da581d39283684d8fcb32b/hpc-test-node3_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W3](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/5312f2682bca598dd47b604ee3fa132d9d475269/hpc-test-node3_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node_W0](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/0ca5039cbe3d314478b4de9fb6f378b6f9d16eca/hpc-test-node_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node_W1](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/3424b7a9a83b0574752c7a97d8ec05588d260752/hpc-test-node_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node_W2](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/593a3a2a0323ec839773ab352908a62d759153a6/hpc-test-node_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node_W3](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/c8bb0d5269bfef47c6614d05bc0a458bda4fd148/hpc-test-node_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/cd1d28d2673bc4afba1a16da65a09bc3e43d7eb9/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/d02427ffb364060c603dcf5ffcea7436109dfa96/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/cbcb4b722803c21a9120f05b657e188828199096/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/f91223b32b24dbe06e130c0990a9152761067e8a/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/7505779bb3387829db2e01f9e121ac61555f787a/vulcan-jenkins_W0) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/363b7dcfa6bdd8e06796e8bfad2b1c299d45c7d5/vulcan-jenkins_W1) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/2d1d489e651c206443a5c20d7477f2b2281cec4c/vulcan-jenkins_W2) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/0aa230e239962b586265d271876a63bf/raw/3cc8d6a614f581fd588ef06867cf24d26475acee/vulcan-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**